R/collect.R

```diff
@@ -11,17 +11,26 @@
 #'  used to filter the predicted values before processing. This tibble should
 #'  only have columns for each tuning parameter identifier (e.g. `"my_param"`
 #'  if `tune("my_param")` was used).
+#' @param type One of `"long"` (the default) or `"wide"`. When `type = "long"`,
+#'  output has columns `.metric` and one of `.estimate` or `mean`.
+#'  `.estimate`/`mean` gives the values for the `.metric`. When `type = "wide"`,
+#'  each metric has its own column and the `n` and `std_err` columns are removed,
+#'  if they exist.
```

Comment:
I didn't have strong opinions here and the original implementation in #689 had the same behavior, so I did it this way, _but_ if we wanted we could have columns like `rmse`, `rmse_n`, `rmse_std_err`.

Reply:
I think excluding them is fine; we can add them later if someone asks for them.